### PR TITLE
Update install.md

### PR DESCRIPTION
**Pull request by JustinN-Replicated**
Initial changes for --ensure-rbac flag; likely need to link to generate-manifests still as well

### CODE REVIEW

**Review by sgalsaleh — COMMENTED** [review]
this flag is also available for upgrades, not just installs. so we should document that as well.

File: content/kots-cli/admin-console/upgrade.md [hunk 1]
```diff
@@ -21,7 +21,9 @@ This command supports all [global flags](/kots-cli/global-flags/) and also:
 | `-n, --namespace` | string | the namespace where the admin console is running _(default "default")_         |
 | `--wait-duration`  | string | timeout out to be used while waiting for individual components to be ready.  must be in [Go duration](https://pkg.go.dev/time#ParseDuration) format (eg: 10s, 2m). |
 | `--with-minio`    | bool   | when set, kots will deploy a local minio instance for storage _(default true)_ |
+| `--ensure-rbac`          | bool   | when set, kots will skip RBAC configuration at upgrade time. (default false) if a role spec is needed, use the [generate-manifests](/kots-cli/admin-console/generate-manifests/) command.                                                         |
```
Comment (sgalsaleh) [inline content/kots-cli/admin-console/upgrade.md:24]:
```suggestion
| `--ensure-rbac`          | bool   | when set, kots will skip RBAC configuration at upgrade time (default false). if custom role specs are needed, use the [generate-manifests](/kots-cli/admin-console/generate-manifests/) command.                                                         |
```

File: content/kots-cli/install.md [hunk 2]
```diff
@@ -44,6 +44,7 @@ This command supports all [global flags](/kots-cli/global-flags/) and also:
 | `--disable-image-push`   | bool   | set to true to disable images from being pushed to private registry                                                                  |
 | `--wait-duration`        | string | timeout out to be used while waiting for individual components to be ready. must be in [Go duration](https://pkg.go.dev/time#ParseDuration) format (eg: 10s, 2m). |
 | `--with-minio`           | bool   | when set, kots will deploy a local minio instance for storage (default true)                                                         |
+| `--ensure-rbac`          | bool   | when set, kots will skip RBAC configuration at install time. (default false) if a role spec is needed, use the [generate-manifests](/kots-cli/admin-console/generate-manifests/) command.                                                         |
```
Comment (sgalsaleh) [inline content/kots-cli/install.md:47]:
```suggestion
| `--ensure-rbac`          | bool   | when set, kots will skip RBAC configuration at install time (default false). if custom role specs are, use the [generate-manifests](/kots-cli/admin-console/generate-manifests/) command.                                                         |
```

**Review by sgalsaleh — APPROVED** [review]

**Comment by JustinN-Replicated** [comment]
Forgot I had left his as a draft 🤦‍♂️ Just double checking before squash/merge

**Review by sgalsaleh — APPROVED** [review]

**Pull request merged**